Buildings/Media/Examples/SteamSaturatedProperties.mo

```diff
@@ -0,0 +1,86 @@
+within Buildings.Media.Examples;
+model SteamSaturatedProperties
+  "Model that tests the implementation of the steam properties at saturated liquid and vapor states"
+  extends Modelica.Icons.Example;
+  package Medium=Buildings.Media.Steam
+    "Steam medium model";
+  parameter Modelica.SIunits.Temperature TMin=273.16
+    "Minimum temperature for the simulation";
+  parameter Modelica.SIunits.Temperature TMax=647.096
+    "Maximum temperature for the simulation";
+  parameter Modelica.SIunits.Pressure pMin=620
+    "Minimum pressure for the simulation";
+  parameter Modelica.SIunits.Pressure pMax=20000000
+    "Maximum pressure for the simulation";
+  Modelica.SIunits.Pressure pSat
+    "Saturation pressure";
+  Medium.Temperature TSat
+    "Saturation temperature";
+  Modelica.SIunits.Conversions.NonSIunits.Temperature_degC TSat_degC
+    "Celsius saturation temperature";
+  Medium.SaturationProperties sat
+    "Medium saturation state";
+  Modelica.SIunits.Density dl
+    "Density of saturated liquid";
+  Modelica.SIunits.Density dv
+    "Density of saturated vapor";
+  Modelica.SIunits.SpecificEnthalpy hl
+    "Enthalpy of saturated liquid";
+  Modelica.SIunits.SpecificEnthalpy hv
+    "Enthalpy of saturated vapor";
+  Modelica.SIunits.SpecificEnthalpy hlv
+    "Enthalpy of vaporization";
+  Modelica.SIunits.SpecificEntropy sl
+    "Entropy of saturated liquid";
+  Modelica.SIunits.SpecificEntropy sv
+    "Entropy of saturated vapor";
+  Modelica.SIunits.SpecificEntropy slv
+    "Entropy of vaporization";
+protected
+  constant Real conv(unit="1/s")=1
+    "Conversion factor to satisfy unit check";
+initial equation
+  Modelica.Utilities.Streams.print("Warning:\n  In " + getInstanceName() +
+    ": This model is a beta version and is not fully validated yet.");
```

Comment:
Somehow this warning seems to have commented all the text below it. I'm telling this by the text color but am not sure. Could you please double check it is correct? Thanks!

Reply:
Strange. It shows up correctly in Dymola (see below). I reformatted it regardless to see if it fixed the text color issue.
![image](https://user-images.githubusercontent.com/44682485/105445975-68b6e600-5c2e-11eb-8116-40d5fd0c3168.png)